selinux/label_linux.go

```diff
@@ -77,3 +77,12 @@ func RestoreContext(aPath string, mode RestoreMode) error {
 
 	return exec.Command("restorecon", args...).Run()
 }
+
+// SnapMountContext finds out the right context for mounting snaps
+func SnapMountContext() string {
+	// TODO: consider reading this from an external configuration file, such
+	// as per app contexts, from
+	// /etc/selinux/targeted/contexts/snapd_contexts like go-selinux and
+	// podman do for container volumes.
+	return "system_u:object_r:snappy_snap_t:s0"
```

Comment:
What is the possible evolution path of this string? When/if we wish to change it (e.g. to `snapd_snap_t` instead of `snappy_snap_t`, what would have to happen? Can you please add the answer as a comment here.

Reply:
This is going to involve quite some changes. First, it's the same problem as now we had with mount units. Those are generated during install. Any later policy changes, that could involve typ change are not tracked by snapd, so we do not know when to regenerate the units. On top of this, remounting a block device with changed security contest, while it's used/mounted already will be stopped by the kernel. That's why we'll need to recommend reboot to transition to the new policy and it'll probably take the same for any future transition.

Reply:
@zyga It is very likely that the policy prefix will not change for a very long, because of the complexity of the type change. I would probably resist it myself, simply because this policy is not just about `snapd`, it's about the entire snappy system.

Reply:
Can this please be added to the code as a code comment.